CodeStarterCamp_Week2/Step2.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  Step2.swift
+//  CodeStarterCamp_Week2
+//
+//  Created by 고혜지 on 1/3/24.
+//
+
+import Foundation
+
+private func generateLottoNumbers() -> Set<Int> {
+  var winningNumbers: Set<Int> = Set<Int>()
```

Comment:
```suggestion
  var winningNumbers: Set<Int> = .init()
```
요렇게 쓰면 쓰기 더 간편할거 같아요! (취향)

Reply:
오오 이런식의 표현법은 몰랐어요